lang/src/org/partiql/lang/visitors/PartiQLVisitor.kt

```diff
@@ -235,6 +235,26 @@ internal class PartiQLVisitor(val ion: IonSystem, val customTypes: List<CustomTy
         insert(target, visit(ctx.value, PartiqlAst.Expr::class), conflictAction, metas)
     }
 
+    // TODO move from experimental; pending: https://github.com/partiql/partiql-docs/issues/27
+    override fun visitReplaceCommand(ctx: PartiQLParser.ReplaceCommandContext) = PartiqlAst.build {
+        insert(
+            target = visitSymbolPrimitive(ctx.symbolPrimitive()),
+            values = visit(ctx.value, PartiqlAst.Expr::class),
```

Comment:
nit: `value` or `values`, that is a question....
I am leaning toward `value` since it is a single expression, but I don't have a strong opinion on this.

Reply:
The _values_ semantically is a bag of values; also unfortunately we've lost the battle already b/c this has been there for quite some time and any change to this will be backward incompatible.